docs/manual/common/guide/production/EnterpriseSuite.md

```diff
@@ -0,0 +1,42 @@
+# Using Lightbend Enterprise Suite with Lagom
+
+Lagom is part of the [Lightbend Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform)—an application development solution for building Reactive systems on the JVM powered by an open source core and the commercial [Enterprise Suite](https://www.lightbend.com/products/enterprise-suite). Lagom services can be developed, deployed and run in production using only open source components, however we recommend considering the advanced features provided by Enterprise Suite.
+
+This page highlights the Enterprise Suite features that are especially useful for most Lagom users, but there are many more. You can find an overview on the [Lightbend web site](https://www.lightbend.com/products/enterprise-suite) or [contact Lightbend](https://www.lightbend.com/contact) to learn more.
+
+## Akka Commercial Addons
+
+Lagom is based on Akka, a set of open-source libraries for designing scalable, resilient systems that span processor cores and networks. [Akka Commercial Addons](https://developer.lightbend.com/docs/akka-commercial-addons/current/index.html) is a suite of useful components that complement Akka and Lagom. These can be included as dependencies of your Lagom services to enable their functionality. It includes:
+
+* [Split Brain Resolver](https://developer.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html)
+* [Multi-DC Persistence](https://developer.lightbend.com/docs/akka-commercial-addons/current/persistence-dc/index.html)
```

Comment:
Not sure about Multi-DC Persistence. I'm that people can understand it as, a drop-in replacement for Lagom Persistence, which is not the case. 

Also, we know that the integration of it with Lagom right now is not that smooth. Lots of extra things need to be taken into account when deploying Lagom in a multi-dc environment.

Reply:
OK I just wanted to list everything included. It's true that it's not a drop-in replacement, though I don't see why you couldn't use it as-is with the Lagom service API. It would be nice if it had better information about how to use it with Lagom, but I'm not sure this is the place for it.

Maybe I should remove the list? Any other opinions?

Reply:
Yes, agree!